lib/Canonicalize/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # This file is distributed under the MIT License. See LICENSE.md for details.
 #
 
+add_subdirectory(SimplifySwitch)
```

Comment:
1. Remove subdirectory
2. Conflate the `*Pipe.cpp` and `*Pass.cpp` files
3. Drop the private header file